src/components/button-group/button-group.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { IButtonGroupProps } from "../../types";
```

Comment:
Might be personal, but I'd say that Component props interface is more useful in the same file as the component definition. Unless the same props are shared by multiple components and there's a reason to make sure that they stay in sync.